.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "vendor"]
 	path = vendor
 	url = https://github.com/istio/vendor-istio.git
+	branch = jaegerzipkin
```

Comment:
indeed, please remove this, I didn't look/know you changed this - you should not
( guess I should add that file to .gitignore too)